### DISCUSSION TITLE
2.1.4

### DISCUSSION THREAD

**Discussion by benhillis** [microsoft/WSL#11234] [General]
* Minor bugs and stabilization changes prior to GA release.

<hr /><em>This discussion was created from the release <a href='https://github.com/microsoft/WSL/releases/tag/2.1.4'>2.1.4</a>.</em>

**Comment by ColorfulRhino** [comment]
Thanks!
The bug that doesn't show the Linux icon in Windows Explorer is still there for me unfortunately. As discusses in https://github.com/microsoft/WSL/issues/10872 and https://github.com/microsoft/WSL/issues/11026

**Comment by lindhe** [comment]
I see that this release is not marked as pre-release in GitHub, like the previous release was:

![image](https://github.com/microsoft/WSL/assets/7773090/a99fa482-6f81-49dd-9321-23a5dff03f93)

But it's still not released, right?

**Comment by matek075** [comment]
Downgrading to this version help to avoid this issue -> `failed to attach disk`
